src/boards/rainbow_esp.h

```diff
@@ -81,7 +89,7 @@ class BrokeStudioFirmware: public EspFirmware {
 	};
 
 	void processBufferedMessage();
-	void processFileMessage();
+	//void processFileMessage(); // No code found ?
```

Comment:
Nice catch. This function was removed one year ago with commit bcf14f7a3b88253915d08044734b9a445195770f

You can safely remove this line.